### ISSUE TITLE
Disabling Select form control does not work

### ISSUE THREAD

**Issue by kormik** [rjaros/kvision#178]
WhenI set the disabled property on Select form control nothing happens and the field is not disabled. For Text form control it works as expected.

Here is a code snippet I used:

```
    @Serializable
    data class FormData(val item: String, val text:String)

    var form1:FormPanel<FormData>

    init {
        form1 = formPanel<FormData> {
            add(
                    FormData::item,
                    Select(label = tr("Item"), options = listOf("A" to "A", "B" to "B", "C" to "C"))
            )
            add(
                    FormData::text,
                    Text(label = tr("Text"))
            )
        }

        button("DISABLE").onClick {
            val select = form1.getControl(FormData::item)!!
            select.disabled = !select.disabled

            val text = form1.getControl(FormData::text)!!
            text.disabled = !text.disabled
        }
    }

**Comment by kormik** [comment]
Actually underlying select HTML element is disabled, but UI component representing the selectbox is not, which leads to confusing behaviour. User can open dropdown, select value, but the value of input field is not changed.

**Comment by rjaros** [comment]
Fixed in 3.13.1

**Comment by kormik** [comment]
Thank you.
